Userland/Libraries/LibJS/Tests/builtins/Date/Date.js

```diff
@@ -24,6 +24,11 @@ test("timestamp constructor", () => {
     expect(date.getSeconds()).toBe(1);
     expect(date.getFullYear()).toBe(1970);
     expect(date.getMonth()).toBe(1); // Feb
+
+    let date = new Date(NaN);
+    expect(date.getTime()).toBe(NaN);
+    let date = new Date(undefined);
+    expect(date.getTime()).toBe(NaN);
```

Comment:
Even if LibJS doesn't complain about it, declaring a variable with `let` in in the same scope multiple times is invalid syntax - don't do it (same below and in other tests)